### PR TITLE
Day 8- Q2: Leaf at same level #260 cpp code added.

### PR DESCRIPTION
**Pull request by Akansha77**
Day 8- Q2: Leaf at same level #260 cpp code added by Akansha77

### CODE REVIEW

**Comment by bh-g** [comment]
The question has already been added to the repo, add your solution in the same folder by creating a file Akansha--C.md

**Comment by Akansha77** [comment]
dii what is the name of the folder in the repo ?

File: Others/Day_8-Q2_Leaf_at_the_same_level/question.md
Comment (bh-g) [inline Others/Day_8-Q2_Leaf_at_the_same_level/question.md:1]: no need to add question

File: Others/Day_8-Q2_Leaf_at_the_same_level/Akansha--C.cpp
Comment (bh-g) [inline Others/Day_8-Q2_Leaf_at_the_same_level/Akansha--C.cpp:1]: Hey, you've named the file correctly:)
However, you don't need to make a new folder, there already exists a folder for this question, pls add the file here-
Day-8/q2: All leaf nodes at same level/

**Review by bh-g — CHANGES_REQUESTED** [review]

**Comment by Akansha77** [comment]
dii changes done

**Comment by bh-g** [comment]
Same as #282

**Pull request closed without merging**